requirements/test.txt

```diff
@@ -1,8 +1,8 @@
 -r base.txt
-pytest==4.0.1
-pytest-cov==2.6.0
-pytest-isort==0.2.1
-pytest-django==3.4.4
+pytest==5.2.2
```

Comment:
@danlipert ok with bumping these versions?

Reply:
definitely not - needs to be removed from this PR @straumer

Reply:
I removed the commit, rebased on master, resolved a small conflict, tested again manually and did a forced push on the branch. It doesn't seem to be updating the PR though, should I make a new PR? @owocki